modules/age.nix

```diff
@@ -216,7 +225,7 @@ in {
       '';
     };
     identityPaths = mkOption {
-      type = types.listOf types.path;
+      type = types.listOf identity;
```

Comment:
Is there a reason we don't use `types.str` rather than making our own type? Probably doesn't make much of a difference but just curious.

Reply:
Can we use str but still check that it doesn't contain any /nix/store strings?

Reply:
Good question -- no idea 😆